API.md

```diff
@@ -1,445 +1,648 @@
+```
+---------------------------------------
+<a name="bucket_exists">
+#### bucket_exists(bucket_name)
+Checks if a bucket exists.
+
+__Parameters__
+
+|Param   |Type   |Description   |
+|---|---|---|
+|``bucket_name``   |*string*   |Name of the bucket. |
+
+__Example__
+
+```py
+try:
+    print(minioClient.bucket_exists("mybucket"))
+except ResponseError as err:
+    print(err)
+```
+---------------------------------------
+<a name="remove_bucket">
+#### remove_bucket(bucket_name)
+Removes a bucket.
+
+__Parameters__
+
+|Param   |Type   |Description   |
+|---|---|---|
+|``bucket_name``   |*string*   |Name of the bucket. |
+
+__Example__
+```py
+try:
+    minioClient.remove_bucket("mybucket")
+except ResponseError as err:
+    print(err)
+```
+---------------------------------------
+<a name="list_objects">
+#### list_objects(bucket_name, prefix, recursive=False)
+Lists objects in a bucket.
+
+__Parameters__
+
+| Param  |Type  | Description  |
+|---|---|---|
+|``bucket_name``   |*string*   | Name of the bucket.  |
+|``objectPrefix``   | *string*   |The prefix of the objects that should be listed. |
+|``recursive``   | *bool*   |``True`` indicates recursive style listing and ``False`` indicates directory style listing delimited by '/'. Optional default is False.   |
+
+__Return Value__
+
+<table>
+    <thead>
+        <tr>
+            <th>Param</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+           object
+            </td>
+            <td> Object</td>
+            <td> Iterator for all the objects in the bucket, the object is of the format:
+            <ul>
+            <li>object.object_name string: name of the object. </li>
+            <li>object.size int: size of the object.</li>
+            <li>object.etag string: etag of the object. </li>
+            <li>object.last_modified datetime.datetime: modified time stamp. </li>
+            </ul>
+            </td>
+            </tr>
+               </tbody>
+</table>
+
+__Example__
+
+```py
+# List all object paths in bucket that begin with my-prefixname.
+objects = minioClient.list_objects('mybucket', prefix='my-prefixname',
+                              recursive=True)
+for obj in objects:
+    print(obj.bucket_name, obj.object_name.encode('utf-8'), obj.last_modified,
+          obj.etag, obj.size, obj.content_type)
+```
+--------------------------------------
+<a name="list_incomplete_uploads">
+#### list_incomplete_uploads(bucket_name, prefix, recursive=False)
+Lists partially uploaded objects in a bucket.
+
+__Parameters__
+
+|Param   |Type   |Description   |
+|---|---|---|
+|``bucketname``   | *string*  |Name of the bucket.|
+|``prefix``   |*string*    |The prefix of the incomplete objects uploaded should be listed. |
+|``recursive`` |*bool*   |``True`` indicates recursive style listing and ``False`` indicates directory style listing delimited by '/'. Optional default is ``False``.   |
+

```

Comment:
Second line in the parameter table should not highlighted fully. Only the params need to be highlighted.